fastapi/openapi/docs.py

```diff
@@ -34,6 +35,10 @@ def get_swagger_ui_html(
         url: '{openapi_url}',
     """
 
+    if swagger_ui_parameters:
+        for key, value in swagger_ui_parameters.items():
+            html += f"{key}: {json.dumps(value)},\n"
```

Comment:
Maybe, we could catch the `TypeError` we will have ifthe value is not serializable in JSON (datetime, object...) to facilitate the dev experience. Here a proposition:

```python
if swagger_ui_parameters:
    try:
        ...
    except TypeError:
        print("swagger_ui values are serialized to Javascript code, please use only basic types like strings, numbers and booleans"")
        raise

Reply:
Good idea, change made.